pkgs/development/python-modules/falcon/default.nix

```diff
@@ -27,26 +38,33 @@ buildPythonPackage rec {
   };
 
   checkInputs = [
+    pytestCheckHook
+    ddt
+    nose
+    pyyaml
+    requests
+    testtools
+    pytest-asyncio
     aiofiles
     cbor2
     httpx
+    jsonschema
+    python-mimeparse
     msgpack
+    mujson
+    orjson
     pecan
-    pytest-asyncio
-    pytestCheckHook
-    pyyaml
-    requests
-    testtools
+    rapidjson
+    ujson
     websockets
+    gunicorn
+    hypercorn
+    uvicorn
```

Comment:
This list was sorted alphabetically before.

Reply:
fixed